prediction/src/views.py

```diff
@@ -81,20 +82,27 @@ def predict(algorithm):
         try:
             predict_method = PREDICTORS[algorithm]
 
-            prediction = predict_method(**payload)
```

Comment:
I think the `**payload` version was a bit cleaner

Reply:
Good point, I've switched to using the kwargs method. In the process it seemed like it would just be simpler to catch all errors in the same way, simply stating the name of the algorithm used and then printing the resulting error. Let me know what you think of this change.

Reply:
My only worry is that by removing the `KeyError` check, the error message may not be as meaningful. I.e., `KeyError: 'dicom_path'` vs `missing parameter 'dicom_path'`.

Reply:
errors from wrong parameters should now be TypeErrors, not KeyErrors, which are a bit more verbose (e.g., the error message specifies whether your call to predict() has an unneeded parameter or an extra parameter). I also added the type of the error to the end of the message, which could be a useful extra piece of information at least during development.